daisy_workflows/build-publish/rhui/rhua_sync_status.py

```diff
@@ -0,0 +1,154 @@
+#  Copyright 2022 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Publish metrics for RHUA sync status."""
+
+import os
+import subprocess
+import time
+
+import requests
+
+
+REPODIR = '/var/lib/rhui/remote_share/symlinks/pulp/content/content'
+
+
+def GetRepodataList(dirname):
+  # Use "find" here because os.walk takes ~30 minutes,
+  # while find takes 10-30 seconds.
+  cmd = ['find', dirname, '-name', 'repodata']
+  output = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+  return output.stdout.split()
+
+
+def PublishMetric(metric_type, points, access_token, mig_name):
```

Comment:
as discussed offline, we either want to use the service library or create a tracking bug to migrate to it in a subsequent pass

Reply:
I'll create a tracking bug.